tools/cldr-code/src/main/java/org/unicode/cldr/draft/FileUtilities.java

```diff
@@ -70,7 +70,7 @@ public static PrintWriter openUTF8Writer(File dir, String filename) throws IOExc
 
     public static PrintWriter openWriter(File dir, String filename, Charset encoding) throws IOException {
         File file;
-        if (dir.equals("")) {
+        if (dir == null) {
```

Comment:
should we also check if it's a file with an empty path?

Reply:
What check would you do?

Reply:
hmmm actually i guess there's no such thing, nvm